account_move_line_sequence/models/account_move_line.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2025 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    sequence_journal_item = fields.Integer(default=16)
```

Comment:
I am wondering if we can get rid of this field altogether, and just use the standard sequence field in the viww

Reply:
@ntsirintanis Do you mean a field named `sequence` already exists, or do you mean something else?

And what would be the benefit of it?

Reply:
Yeah, a field sequence already exists in v16 account.move.line.  The benefit would be that we won't be introducing a new field in the model, just add it to another view. However, I think I will ditch this idea because this is more clear anyway.